### ISSUE TITLE
Add option to get version bump info

### ISSUE THREAD

**Issue by Noviny** [Noviny/changesets#10]
Something like: `build-releases analyze` that should output the versions of what will be published if it is run now. Possible formats of output are:

```
| Package Name | Bump Type | New Version  |
|:-------------:|:-------------:|:-------------:|
|    packageA    |       minor       |      1.1.0           |
|    packageB    |       major       |      1.0.1           |
|    packageC    |       patch       |      2.0.0          |
```

Similar, make sure there is a script we can call not from the command line, so we can build extension tools on top of this. #5 would very much like this.

Info I don't think we care about showing - what is being bumped as a dependency vs being a main changeset item.

**Comment by Noviny** [comment]
This should read all changesets to output this

**Comment by Noviny** [comment]
Released as of `@changesets/cli 1.0.1`
